guide/results-viewer-react/package.json

```diff
@@ -4,7 +4,8 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "dev": "npm run lint && webpack serve --port 3001 --mode development --open --hot",
+    "dev-lint": "npm run lint && webpack serve --port 3001 --mode development --open --hot",
+    "dev": "webpack serve --port 3001 --mode development --open --hot --watch",
```

Comment:
You could just remove dev-lint. and then add lint to the build script. So on mac it won't hot refresh if we have the npm run lint?

Reply:
It was giving an error for Bryan yesterday, but I just double-checked and it runs for me just fine. Still, it would be nice to have a non-linted option. I like the idea of moving the lint to the build, so I'll do that. I also added the `--watch` flag.